pkg/testing/environment.go

```diff
@@ -26,10 +26,14 @@ import (
 	"testing"
 
 	"github.com/pulumi/pulumi/pkg/util/fsutil"
-	"github.com/pulumi/pulumi/pkg/workspace"
 	"github.com/stretchr/testify/assert"
 )
 
+const (
+	bookkeepingDir              = ".pulumi"
+	pulumiCredentialsPathEnvVar = "PULUMI_CREDENTIALS_PATH"
```

Comment:
I had to break a circular dependency in order to use the test environment in some new tests that depend on `workspace`.